src/main/java/ArrayMinHeap.java

```diff
@@ -26,7 +26,41 @@ public ArrayMinHeap(int initialCapacity) {
 
     @Override
     public boolean add(T element) {
-        return false;
+        if (size() == heap.length) {
+            expandCapacity();
+        }
+        heap[size] = element;
+        bubbleUp(size);
+        size++;
+        return true;
+    }
```

Comment:
Idk what the `return false` case would be here.

Reply:
There isn't always a `false` case. `add` returns true if the item was added, if the collection was changed. In this case, the collection _always_ changes, the element can always be added, so `return true`.